packages/@lwc/integration-karma/test/static-content/x/staticUnsafeTopLevel/staticUnsafeTopLevel.html

```diff
@@ -27,5 +27,4 @@
   <template if:true={doRender}>
     <tr></tr>
   </template>
-  </template>
```

Comment:
As it turns out, we accidentally made this mistake, and I discovered it in our Karma tests. 😆